apps/quotes/api/views.py

```diff
@@ -24,4 +24,5 @@ class QuoteAPIView(RetrieveAPIView):
     """API view for a single occurrences."""
 
     queryset = Quote.objects.all()
+    lookup_field = 'slug'
```

Comment:
Do we want this specified? IIRC it is inferred from the route parameter by default.

Reply:
IIRC, it didn't work with the `slug` field until I specified `lookup_field` (after searching on StackOverflow)... But I think we'll get rid of this view soon anyway (to use GraphQL consistently).